### ISSUE TITLE
Repository addon is automatically installed by daemon in background

### ISSUE THREAD

**Issue by 9p6** [elgatito/plugin.video.elementum#975]
I specifically chosen to install elementum from zip file in order to *not* have the [repository](https://github.com/ElementumOrg/repository.elementumorg/releases/download/v0.0.2/repository.elementumorg-0.0.2.zip) addon installed but it seems elementum daemon ~~downloads~~ [generates](https://github.com/elgatito/elementum/blob/master/repository/repository.go#L122) and installs it by itself.

```
warning <general>: Repository add-on repository.elementumorg uses plain HTTP for add-on downloads in path http://127.0.0.1:65223/ - this is insecure and will make y
our Kodi installation vulnerable to attacks if enabled!
```

I know that warning is for `localhost` but nobody asked me and overall it feels quite invasive.

**Comment by antonsoroko** [comment]
@9p6 JFYI: new version will have a setting for that https://github.com/elgatito/plugin.video.elementum/commit/05c490b975f504f025d89534da29bcf6e6261186

although, personally i do not see much value it this. e.g. if you want to disable updates - you can do with or without repo.
but vast majority of users needs auto update, thus they need repo.

>in order to not have the repository addon installed

can you please describe why this can be useful/important?

**Comment by 9p6** [comment]
I found Elementum disappointing and I'm no longer interested in it. I didn't like the fact that it made changes to my system and created another addon. I also didn't like the repo addon in itself - it had some dynamic stuff in it, wtf is that? I also didn't like that Elementum opened so many ports and autostarted HTTP server and showed me some shitty messages on reboot. I don't want any of this stuff. Long story short I just deleted and moved on.

```
